Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -887,16 +887,20 @@ public void UnsubscribeFromPrimaryPointerChanged(PrimaryPointerChangedHandler ha
         /// <returns>Pointer Data if the pointing source is registered.</returns>
         private bool TryGetPointerData(IMixedRealityPointer pointer, out PointerData data)
         {
+            Profiler.BeginSample("[MRTK] FocusProvider.TryGetPointerData");
```

Comment:
How noisy do markers like this end up being? There are some methods in here that I'm not sure how useful profiling them would be. I feel like putting in too many markers could blow up the profiler's list and become harder to be useful.

Reply:
not sure. good question

Reply:
let's back them off until we know they tell us somethin